libfwupdplugin/fu-common-version.c

```diff
@@ -545,3 +549,41 @@ fu_common_vercmp_full (const gchar *version_a,
 	}
 	return fu_common_vercmp_safe (version_a, version_b);
 }
+
+/**
+ * fu_common_version_kernel:
+ * @minimum_kernel: The minimum kernel version to check against
+ * @error: (nullable): optional return location for an error
+ *
+ * Determines if the system is running at least a certain required kernel version
+ *
+ * Since: 1.6.2
+ **/
+gboolean
+fu_common_version_kernel (const gchar *minimum_kernel, GError **error)
+{
+#ifdef HAVE_UTSNAME_H
+	struct utsname name_tmp;
+
+	memset (&name_tmp, 0, sizeof(struct utsname));
+	if (uname (&name_tmp) < 0) {
+		g_debug ("Failed to read current kernel version");
+		return TRUE;
```

Comment:
TRUE? Wouldn't False be more appropriate here if we're checking for something that requires a specific kernel version?

Reply:
yeah I'll adjust.  I'm not sure why we had it like that previously in thunderbolt.